src/py42/services/users.py

```diff
@@ -330,3 +330,42 @@ def remove_role(self, user_id, role_name):
         role_name = quote(role_name)
         uri = u"/api/UserRole?userId={}&roleName={}".format(user_id, role_name)
         return self._connection.delete(uri)
+
+    def update_user(
+        self,
+        user_id,
+        username=None,
+        email=None,
+        password=None,
+        first_name=None,
+        last_name=None,
+        note=None,
+        quota=None,
+    ):
+        """Updates an existing user.
+        `REST Documentation <https://console.us.code42.com/apidocviewer/#User-put>`__
+
+        Args:
+            user_id (int): An ID for the user.
```

Comment:
This looks like the `userUid`.  In the users service, we still refer to `user_uid` since that will match the response, and we don't want to confuse with the legacy ID.

Reply:
the REST endpoint can accept user_uid or user_id, I used user_id for consistency with the other methods in py42, would you like me to change it to user_uid instead?

Reply:
I think we tend to use `user_uid` whenever possible, so yes I would prefer it there.  The methods that located _near_ this method happen to use the legacy ID because they have to, but other methods in this service use the `user_uid`.

Reply:
Obligation _or_ mistake, in case there are any that slipped through when not necessary

Reply:
Makes sense. I've updated it to use uid.